developer_docs/testing.md

```diff
@@ -485,14 +485,14 @@ afterEach(() => server.resetHandlers());
 afterAll(() => server.close());
 ```
 
-If the component makes use of the `formatDate` util, some of the functions in that rely on the `client/i18n.js` file that also makes an AJAX request, which sometimes leads to an `ERRCONNECTED` error on the console, even though your tests pass. You can fix it by adding a mock for that specific i18n file:
+If the component makes use of the `formatDate` util, some of the functions in that rely on the `client/i18n.js` file also make an AJAX request, which sometimes leads to an `ERRCONNECTED` error on the console, even though your tests pass. You can fix it by adding a mock for that specific i18n file:
```

Comment:
I interpreted the updated phrasing to state that the functions themselves are also making an AJAX request, as opposed to the original docs, which I think implies that the `client/i8n.js` file is making AJAX requests in tandem with the functions. 

However, I'm not entirely certain and feel that this entire sentence could possibly be made clearer, let me know what you think!